src/integrationTest/java/uk/gov/hmcts/reform/judicialbooking/config/SwaggerPublisher.java

```diff
@@ -20,19 +21,24 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
  */
-@SpringJUnitWebConfig
-@SpringBootTest
-@AutoConfigureMockMvc
-class SwaggerPublisher {
+public class SwaggerPublisher extends BaseTest {
 
-    @Autowired
-    private transient MockMvc mvc;
+    private MockMvc mockMvc;
+
+    @Inject
+    private WebApplicationContext wac;
+
+    @Before
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+        MockitoAnnotations.openMocks(this);
+    }
 
     @DisplayName("Generate swagger documentation")
     @Test
-    @Disabled
-    void generateDocs() throws Exception {
-        byte[] specs = mvc.perform(get("/v2/api-docs"))
+    //@Disabled
+    public void generateDocs() throws Exception {
+        byte[] specs = mockMvc.perform(get("/v3/api-docs"))
```

Comment:
Can we use constant for URL: `org.springdoc.core.Constants.DEFAULT_API_DOCS_URL`

Reply:
Resolved in #739 see 472abc738e825ab97ea64988a66880e05b08835f